bats/tests/compose/testdata/app/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.9-slim
```

Comment:
Not sure how to deal with this, but we make almost all images configurable, so tests can pull them from `ghcr.io` instead of Docker Hub, to avoid the pull limits.

So ideally the image name should be passed in to the the Dockerfile via an environment variable the the test could export.

Maybe we should fix this in a follow-up PR, as there will be several pieces to it.